lib/util/pretty-time.ts

```diff
@@ -40,3 +41,44 @@ export function toMs(time: string): number | null {
     return null;
   }
 }
+
+const rangeRegex = regEx(/^(?<operator>(>=|<=|<|>))\s*(?<age>.*)/);
+
+export function satisfiesDateRange(
+  date: string,
+  range: string
+): boolean | null {
+  const grps = range.trim().match(rangeRegex)?.groups;
+  if (!grps) {
+    return null;
+  }
+
+  const { operator, age } = grps;
+  const luxonDate = DateTime.fromISO(date);
+  if (!luxonDate.isValid) {
+    logger.debug(`Invalid date specifier: '${date}'`);
+    return null;
+  }
+
+  const dateMs = luxonDate.toMillis();
```

Comment:
we need to convert to UTC before otherwise we probably compare different time zones